views/login.ejs

```diff
@@ -6,6 +6,9 @@
 <body>
   <% include ../views/partials/tab %>
   <h2>Login</h2>
+  <% if (message.length > 0) { %>
+    <p class="flash"><%=message %></p>
```

Comment:
Hey @tarebyte - right now the flash takes the entire width of the page. Can you help me with some styling?